docs/conf.py

```diff
@@ -25,8 +25,13 @@
     'sphinx.ext.napoleon',
     'sphinx.ext.doctest',
     'sphinx.ext.mathjax',
+    # 'sphinx_automodapi.automodapi',
+    # 'sphinx_automodapi.smart_resolver'
```

Comment:
Why were these commented out?

Reply:
Oh, an error was thrown for me locally while trying to run `tox -e build_docs` on my machine, so I comment them out for now.